src/types/siman.rs

```diff
@@ -0,0 +1,507 @@
+/*
```

Comment:
I think this is supposed to be a module comment, so you need to replace `/*` with `/*!`.